src/react/ViewReports/ViewReportsWrapper.tsx

```diff
@@ -37,7 +37,7 @@ const ViewReportsWrapper: FC<ViewReportsWrapperProps> = (viewReportProps) => {
         <QueryClientProvider client={queryClient}>
           <ErrorBoundary fallback={<ViewReportsError onBackButtonClicked={viewReportProps.onBackButtonClicked} />}>
             <Suspense fallback={<ViewReportSkeleton {...viewReportProps} />}>
-              <div className="absolute top-0 left-0 right-0 bottom-0 bg-white z-[50]">
+              <div className="absolute top-0 left-0 right-0 bottom-0 bg-white z-50">
```

Comment:
did you check that this works with the `ViewAllReport` view? the stacking context gets weird there

Reply:
the value isn't changed, z-[50] -> z-50. z-50 is a thing
![Screenshot 2025-06-03 at 4 24 54 PM](https://github.com/user-attachments/assets/b918d956-c6a8-4dfa-85b1-66f400a1a0a7)